lib/_http_incoming.js

```diff
@@ -270,11 +270,11 @@ IncomingMessage.prototype._addHeaderLine = _addHeaderLine;
 function _addHeaderLine(field, value, dest) {
```

Comment:
Shouldn't the comment block describing the behavior of this function be updated as well to describe this new behavior?